### DISCUSSION TITLE
error for lauching app video

### DISCUSSION THREAD

**Discussion by ghost** [elementary/videos#313] [Q&A]
hello the community !

Today, i would like open the aplication "video" but it's impossible i have this problem on the screen image of my computer. See my screen.  I’ve already done a resettlement. can I get some help.

Thank you 

![Capture d’écran de 2022-08-04 11-24-24](https://user-images.githubusercontent.com/22295927/182812916-c673ddaa-8910-450b-89e4-01f4afdb23d7.png)

**Comment by jeremypw** [comment]
Try entering `flatpak run io.elementary.videos ` in a terminal and post what messages appear.  Flatpaks can be a bit slow to start when first run (depending on the power of your machine) but should not take more than a few seconds.

**Reply by ghost** [reply]
I have this result on my terminal 

```
~$ flatpak run io.elementary.videos
Gtk-Message: 11:35:37.178: Failed to load module "canberra-gtk-module"
Gtk-Message: 11:35:37.178: Failed to load module "canberra-gtk-module"

(io.elementary.videos:2): GLib-CRITICAL **: 11:35:37.179: g_strsplit: assertion 'string != NULL' failed
~$ flatpak run io.elementary.videos
```

**Reply by ghost** [reply]
I specify that I did not make any changes to the module. this morning there was an update of the system, maybe the problem comes from there

**Comment by jeremypw** [comment]
Hmm - are you on EOS 6.1? When you say "resettlement" do you mean that you have uninstalled and reinstalled `io.elementary.videos` ? 

I am on EOS7 at the moment but I'll switch to 6.1 and check there has not been a faulty release.

**Reply by ghost** [reply]
I think I’ll wait for the correction to appear in the next update. But it seems that even after a resettlement the worry persists. I will find out for myself maybe the change of version will solve the problem but I prefer not to fiddle too much with the operating system. Thanks anyway for the answer, I think I’ll wait but yes i am in 6.1

**Comment by jeremypw** [comment]
I am trying to reproduce your issue on 6.1 - would you mind posting the output of `flatpak list | grep -i videos` and `apt list --installed | videos` so that I can be sure what you have installed?  Thanks.

**Comment by ghost** [comment]
```
Vidéos io.elementary.videos	2.8.4	stable	appcenter	system

apt list --installed | videos

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

videos : commande introuvable

```

**Comment by jeremypw** [comment]
Sorry the second command should have been `apt list --installed | grep -i videos`.  But I have now tried installing io.elementary.videos flatpak version 2.8.4 on EOS6.1 system-wide after uninstalling all other videos packages.  It runs without a problem.

So presumably there is something else about your system that is exposing a hidden bug in the code of that package (?).  Unfortunately trying to debug flatpak packages remotely is a little complicated.  I have had a look at the code and the function giving the error you mentioned is only used a couple of times - but in both cases there is no obvious way a `null` string could be used.

To get more information you need to open a shell inside the flatpak sandbox and run the app there with `gdb` to get a full backtrace of the error.  If you are interested in trying then there are instructions here: https://blogs.gnome.org/mclasen/2017/01/20/debugging-a-flatpak-application/

**Reply by ghost** [reply]
Thank you for the help and for all this information, I trying your solution and i will come back later if I find the solution with the content you sent me. Waiting for a good day
